src/commands/Minion/minion.ts

```diff
@@ -66,6 +66,71 @@ export default class extends BotCommand {
 		return msg.send(msg.author.minionStatus);
 	}
 
+	async ironman(msg: KlasaMessage) {
+		if (!msg.author.hasMinion) {
+			throw hasNoMinion(msg.cmdPrefix);
+		}
+
+		if (msg.author.minionIsBusy) {
+			return msg.send(msg.author.minionStatus);
+		}
+
+		if (msg.author.ironman) {
+			await msg.send(
+				`Would you like to stop being an ironman? You will keep all your items and stats but you will have to start over if you want to play as an ironman again.Please say \`yes\` to confirm.`
+			);
+			try {
+				await msg.channel.awaitMessages(
+					answer =>
+						answer.author.id === msg.author.id &&
+						answer.content.toLowerCase() === 'yes',
+					{
+						max: 1,
+						time: 15000,
+						errors: ['time']
+					}
+				);
+				await msg.author.settings.sync(true);
+				await msg.author.settings.update(UserSettings.Minion.ironman, false);
+				await msg.send('You are no longer an ironman BTW.');
+			} catch (err) {
+				return msg.channel.send('Cancelled ironman swap.');
+			}
+		}
+
+		await msg.send(
+			`Are you sure you want to start afresh and play as an ironman? (Reset your bank and all your stats, lose the ability to get GP from dailies,trade,dice,duel,etc) Please say \`yes\` to confirm.`
```

Comment:
I think we should make a much bigger warning and list specifically everything that it will reset, e.g. bold text and  ⚠️ emoji.